### PR TITLE
Edits to Big Animal PR5154

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

**Comment by ebgitelman** [comment]
If possible, it would be better for me to edit this much new content before it's released. Let's try to coordinate that in the future.

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/datadog.mdx [hunk 1]
```diff
@@ -2,63 +2,63 @@
...
 
-1. Select an existing project from the **Projects** page.
+1. From the Projects page, select an existing project.
 2. Go to **Settings** on the left-side navigation.
-3. Select **Integrations** from the **Settings** drop down list.
-4. Select **Datadog**, a window pops-up to provide the details: 
-   - **Datadog API Key** &mdash; Provide an API Key. For more details, see [Datadog API Key](https://docs.datadoghq.com/account_management/api-app-keys/). The API Key is sensitive and can't be retrieved once saved.
-   - **Datadog Site Name** &mdash; Provide the Datadog site name by selecting from the available options in the drop-down list. The telemetry goes to this chosen Datadog site. 
-   - **Datadog Site URL** &mdash; The Datadog site URL gets selected automatically based on the Datadog site name you selected. 
-   - **Datadog API Key ID** &mdash; Provide the API Key ID, that acts as the name to store an identifier for the key. This identifier is used for future reference to check the configuration details.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/datadog.mdx:26]: I could not make sense of this bullet item. Please check my edit below.

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/newrelic.mdx [hunk 2]
```diff
@@ -5,57 +5,58 @@ title: "New Relic"
...
-A pre-existing New Relic account is required to use this integration.
+You need a New Relic account before you can use this integration.
 
 ## Enable New Relic
 
 To enable the **New Relic** integration: 
-1. Select an existing project from the **Projects** page.
+
+1. From the **Projects** page, select an existing project.
 2. Go to **Settings** on the left-side navigation.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/newrelic.mdx:20]: I don't understand this step.

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/datadog.mdx [hunk 3]
```diff
@@ -2,63 +2,63 @@
...
+You need a Datadog account before you can use this integration.
 
 ## Enabling Datadog
 
-To enable the **Datadog** integrations: 
+To enable the Datadog integrations: 
 
-1. Select an existing project from the **Projects** page.
+1. From the Projects page, select an existing project.
 2. Go to **Settings** on the left-side navigation.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/datadog.mdx:20]: Don't understand this step. Is it separate from the next step?

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/newrelic.mdx [hunk 4]
```diff
@@ -5,57 +5,58 @@ title: "New Relic"
...
-3. Select **Integrations** from the **Settings** drop down list.
-4. Select **New Relic**, a window pops up to provide the details: 
-   - **New Relic API key** - Provide an API Key. For more details, see [New Relic API Key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/). The API Key is sensitive and can't be retrieved once saved.
-   - **New Relic API Key Name** - Provide the API Key name to store an identifier for the key. This identifier is used for future reference to check the configuration details.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/newrelic.mdx:23]: I similarly had issues with this description. See whether my edit makes sense.

**Review by nidhibhammar — APPROVED** [review]
LGTM

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/newrelic.mdx [hunk 5]
```diff
@@ -5,57 +5,58 @@ title: "New Relic"
...
+Generate a new API license key in New Relic to enable this integration in BigAnimal. This API key must be specific to this integration. Don't share the key. Whenever you disable this integration, revoke the API key using the New Relic interface. Revoking the key disables the New Relic ingestion and billable usage from the BigAnimal integration. Revoking the key doesn't impact the other services that use the New Relic API.
 !!!
 
 ## Metrics
 
-BigAnimal sends a subset of the OpenTelemetry Collector's metrics from the hostmetrics and kubeletstats receivers, plus BigAnimal custom metrics for the monitored Postgres instance. 
+BigAnimal sends a subset of the OpenTelemetry Collector's metrics from the hostmetrics and kubeletstats receivers, It also sends BigAnimal custom metrics for the monitored Postgres instance.
```
Comment (dwicinas) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/newrelic.mdx:37]: Need a period after "receivers."

**Review by nidhibhammar — APPROVED** [review]

**Pull request merged**